gstools/field/base.py

```diff
@@ -97,10 +97,10 @@ def __call__(
         pos : :class:`list`
             the position tuple, containing main direction and transversal
             directions
-        field : :class:`numpy.ndarray`
-            the field values.
-        mesh_type : :class:`str`
-            'structured' / 'unstructured'
+        field : :class:`numpy.ndarray` or :any:`None`, optional
+            the field values. Will be all zeros by default.
```

Comment:
Better phrase "will be np.zeros_like(pos) if None given"?

Reply:
Problem is, that `np.zeros_like(pos)` is not true. Neither for unstructured meshes (`np.zeros_like(pos[0])`, can even be different in 1D) nor structured meshes (`np.zeros(tuple(map(np.size, pos)))`). So I think *"Will be all zeros"* is quite descriptive at this point.

Reply:
My main point was actually the "by default". I'm just not sure if that could be confusing if handing over your own field.
And yeah, my `np.zeros_like(pos)` is some kind of pseudo code.

Reply:
Ok, so I will replace the "by default" with "if None is given".